lib/modules/dosomething/dosomething_home/dosomething_home.module

```diff
@@ -53,7 +53,7 @@ function dosomething_home_preprocess_node(&$vars) {
       }
 
       $tiles['status'] = TRUE;
-      $tiles['image'] = '<img src="' . $tiles['image_url'] . '">';
+      $tiles['image'] = '<img src="data:image/gif;base64,R0lGODlhAQABAIAAAAUEBAAAACwAAAAAAQABAAACAkQBADs=" data-src="' . $tiles['image_url'] . '">';
```

Comment:
Aye. It's a placeholder 1x1 spacer gif so that we can lazy-load in the images without flashing a broken image tag.